### DISCUSSION TITLE
Color settings don'show excactly the really chosen color... sometimes!

### DISCUSSION THREAD

**Discussion by ANDREAPIPPI** [NightscoutFoundation/xDrip#3176] [Bug reports]
I've been seeing this for some time: if I choose the color yellow, just a little transparent, for example for "Treatment/Prediction curves/Insulin on board", I see an almost black color in the preview window. The same in the list, but in the graph the color is correctly yellow.
Also the circles that show the shade, lightness and transparency that we are choosing, once
they were black and clearly visible, but now they are white and almost invisible. (Image 1).

![Color Settings-1](https://github.com/NightscoutFoundation/xDrip/assets/32884006/86c23099-f3d5-4897-8702-66870218884f)

The strange thing is that the incorrect color does not happen in all cases.
For example, in "Glucose Lines and Values", the "high" and "low" colors are shown well, both in the settings and in the graph, but the "in range" values are shown incorrectly in the settings and shown correctly in the graph .
In the past this did not happen, what was chosen in the settings was the same on the graph (Image 2).

![Color Settings-2](https://github.com/NightscoutFoundation/xDrip/assets/32884006/d204e80f-b328-4581-8281-e42da7ec9134)

I'm curious to know if it's my problem or common to other users.Bye.
Andrea  :-)

**Comment by Navid200** [comment]
This may be phone-related.

I just tried the latest xDrip Nightly on Pixel 6a.  I cannot reproduce any of the problems you have described.

**Comment by ANDREAPIPPI** [comment]
You are right, I changes system color from black to white, and now everything works.
BTW I have a POCO F3 with Latest Nightly and MIUI 14 based on Android 13, I suppose XIAOMI's. 
May be someone else with POCO, OPPO or XIAOMI devices have the same problem but still unnoted....
Thank you, NAVIDS200
Andrea    :-)
